python/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "lancedb-py"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/lancedb/lancedb"
+keywords = ["lancedb", "lance", "database", "search", "python"]
+categories = ["database-implementations"]
+
+[lib]
+name = "lancedb"
+crate-type = ["cdylib"]
+
+[dependencies]
+arrow = { version = "43.0.0", features = ["pyarrow"] }
+arrow-array = "43.0"
+arrow-schema = "43.0"
+pyo3 = { version = "0.19", features = ["extension-module", "abi3-py38"] }
+vectordb = { path = "../rust/vectordb" }
```

Comment:
The unfortunate thing here is we'll have two copies of the lance Rust library, so the install size of `lancedb` python is going to nearly double.

Reply:
The plan is eventually we will just remove the pylance dependency from lancedb python package.